graphdatascience/query_runner/progress/static_progress_provider.py

```diff
@@ -0,0 +1,27 @@
+from typing import Optional
+
+from .progress_provider import ProgressProvider, TaskWithProgress
+
+
+class StaticProgressStore:
+    _progress_store = {}
+
+    @staticmethod
+    def register_task_with_unknown_volume(job_id: str, task_name: str):
+        StaticProgressStore._progress_store[job_id] = TaskWithProgress(task_name, "n/a")
+
+    @staticmethod
+    def get_task_with_volume(job_id: str) -> TaskWithProgress:
+        return StaticProgressStore._progress_store[job_id]
+
+    @staticmethod
+    def contains_job_id(job_id: str) -> bool:
+        return job_id in StaticProgressStore._progress_store
+
+
+class StaticProgressProvider(ProgressProvider):
+    def root_task_with_progress(self, job_id: str, database: Optional[str] = None) -> TaskWithProgress:
+        if not StaticProgressStore.contains_job_id(job_id):
+            raise ValueError(f"Task with job id {job_id} not found in progress store")
```

Comment:
I checked this error type (that we use a lot in this repo) and I don't think it's correct to use it here. I think instead let's just use `Exception`
```suggestion
            raise Exception(f"Task with job id `{job_id}` not found in progress store")
```

Reply:
In fact, it's not correct to use in many of our current usages.